cost/aws/unused_rds/README.md

```diff
@@ -27,6 +29,8 @@ This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Auto
                   "cloudwatch:GetMetricStatistics",
                   "cloudwatch:GetMetricData",
                   "cloudwatch:ListMetrics",
+                  "rds:DescribeDBInstances",
+                  "rds:ListTagsForResource"
```

Comment:
No, all of them are on that list, they were tested by running this policy in a customer tenant

Reply:
@vadgun - which customer tenant?

Reply:
This policy is not looking at DB Clusters (which requires permission `rds:DescribeDBClusters`) -- it's only currently looking at RDS DB **_Instances_**.  

I would assume that majority of RDS deployments today are actually "DB **_Clusters_**" -- all Aurora RDS, and non-Aurora if single AZ == "DB Clusters" instead of a DB Instance

Reply:
> @vadgun - which customer tenant?

Hi @nia-vf1, i just copy paste, what you say in policy channel support on Teams.
<img width="606" alt="image" src="https://github.com/flexera-public/policy_templates/assets/22282464/fdeee695-1c0b-49c1-a29a-e8eea889cc13">

Reply:
> 
Hi @bryankaraffa 
I need to add rds:DescribeDBClusters to the list and CFT File?  
Thank you

Reply:
Hi @bryankaraffa , @nia-vf1, How i can test this on staging, or how i can see the right permissions used, is there one left ?

Need to change rds:DescribeDBClusters instead of "rds:DescribeDBInstances", at the list and CFT File? or is fine right now?

Reply:
@vadgun  regarding the right permissions used, I don't know if there is one left or if we now have them all. I would have to test this on my own account but don't have the cycles today.

Oleksiy should know as I think he originally made the change to the policy that requires the new permissions - https://github.com/flexera-public/policy_templates/pull/1326 

@night-codes which new AWS IAM permissions are required to now run this policy ?

@vadgun on a separate note, keep rds:DescribeDBInstances in the CFT

Reply:
Hi @nia-vf1 should erase this 
 - `sts:GetCallerIdentity`
 because in the policy there is a GET request from AWS to this Action 
 <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAA+oAAAA+CAYAAAC82eF2AAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAAFiUAABYlAUlSJPAAACxaSURBVHhe7Z0JvIzVG8cfJISyL1GKkKVE+77vuyyJVFpJWlTWhGhRpGgvSaRNSbvUv00qa1nSlTWSPUu2wv/9nplzvXfMvXfmcm+Tfl+f92PuzDvvnPcsz3m2c9582wJMCCGEEEIIIYQQKUH+6P9CCCGEEEIIIYRIAWSoCyGEEEIIIYQQKYQMdSGEEEIIIYQQIoWQoS6EEEIIIYQQQqQQMtSFEEIIIYQQQogUQoa6EEIIIYQQQgiRQshQF0IIIYQQQgghUggZ6kIIIYQQQgghRAohQ10IIYQQQgghhEghZKgLIYQQQgghhBApRL5tAdHXQgghxH+Tpw6PvvgP0npi9IUQQgghUgVF1IUQQgghhBBCiBRChroQQgghhBBCCJFCyFAXQgghhBBCCCFSiBytUf9t8e+2cNFv7nX1g6payRIl3GuI99nsOXNtxcpV7r3MKFRoT6tWrarNnj3HNm3aHH03PqVLlbRqVQ+0P/9cbz+nzbK/t2yJfhJh34oVrHy5slawYMHoOyJZVqxcaT/8OM1mz51nBQrkt7q1alm9Q+sG7VQoeoZlWv+eypX2dW2RaPvXOrim7RnTZpOm/Gjdez1gRzQ4zO5uf5sVDv3+f5EtQV0/+ewL9smn/7OrWjSzyy65KPrJjvwSjKV7eva2EvvsYz27dbayZcpEPxFi9ySr8fHuBx/Zc4Nesn322dsOqVPbfVb1wAMsX758kRNSaI36T8vNLhm0xFau2WAvttzfLqiRyz71//ga9Y2bNlmfvv1twqQp0XfM9i5e3O67t4sdeECV6DsR/JwU5tyzzrDWN1wb/St1mTtvvt3To7d7He/e/klQRT8cPcYGDX7ZzjjtFGt19ZU76AO5Ab/7/YSJNuilYbb499+tVKBfNr70YjvzjNPy5Pdzi3+qPoUQu5YcGepe4YHuXTtZg8MOda8h3mdPBYoTAiMrMOo63HGbPdSvf7qhnxl+UvSTzpq1a6OfbGevvfayK5s1sXOCcwsUKBB997/Hgl8XWtcevWzrlq0JTcyb//rL3nzrHXvjrZFO6Q2D0+WOW2+2eofUdX9nVf9wfaur7MLzzkm4/e/v0c1KlNgn+k6EZA11nAcffDzaKVxtgj5SZf/9op/syNTpM+y9Dz52Tp1WV7WIvpvaMDZozwL5CwTGdyertO++0U8ywrB+aehwe+udd61Rw4uDsdB0u0GSBPSHzz7/0sYEhs+VVzS1eoceEv1EiNQjq/ERnpsgf/78dtH55zqD3s0RIUP90zlbrfHz82zVH3/YHnvsYc9eU8euqb9z88jwqVts4NdrrFKJgvZ6s2LRd+Mz4Nu/rd2QH93rC46pbu+2LO5e5xo5MNSRDd99P8Heefd959BlvqhYoYKT+aefepIVKVIkembiJCNv1q5bZ6Pe+8A+/3KsLVm61LUhzuTrr7kqaPeK0bMS499iqCc652ZGXhvqyczH6zdssN4P9bWp06ZnWveQzDUTYey4b+3hRwfY1q1bo+9EoP81Cgx25s3X3nzbhr36ulUoX856db/HypXd7vT+K+iz/R5/0l2n4cUXOnmSk7l2V5Noff6TrFy5ykaMHGVfjv3GVq9e4/T244892po3bewcJmE4d9hrbwT1/J2tX7/efX7OmafbpRddkCGABBuCe0fnfH3ESHdurJ3iQU+aE8iuFwa/bNNm/LRLAkLJlBOZOXHyDzbq/Q/sl9lz3fmQWXkTIZF7/+OP1db53p5xba3MdHHxz7HbpL4zwFHKEKAoYHTQ514cYh+P+Sx6xn+TtcGEvj6Y2BIBoYFx9+obI9xr6pH6TK/TQABs3LjRCbdUZemyZYHi+IEtCgRQ7MQby9djx9m47763TYGS9m/hu/ET3ERw4vHH2r4VM1dGl69Y4c5lkjjt5JNyrDjQd1CG036ZbVu26gERIrXJanxgwLzzxis26OmBdtbpp7r3Rr3/YWDofe1eh/ly3t/OSC8WKLh///23jU7bGMiT6Ic55JXJ6+2bqXNtw1/Zj6MzDtrDahxQycqUKmUtDi8afTd1QKm+v0/fwMB5PCIbok5dIpLPDhpsY/73hfs7WRKVNxict93ZyRlQGOlAGSYFSu8Lg4c4wzsZUMy7de5go94cbo898qAzbDIDpZfzODDQReYkMx8XKVzYGWlEfY8+6girWLFC9JOMJHPN7KCffBr0Va7TqmULe/u1oXb9NS2dvvPt9xNs3Z9/uvPq1zvEOZ6WLltuCxcucu95VgVGz+w5c9x3Dq1bJyWMdEi0Pv8p5s1fYLff3dk5UJEngN5ONtR9Dz5sq1b94d4DjOk7OnZxn3ljFjn/ymtv2iP9B6aPd3Q5gi/XtWlng19+Jf3cWNBhf1240Lr1vN/ad+zqjPRdQaLlBO75gYf7Wa/gXn+cOj3TsiZKovcu/n3kiaGOp9lPbHh3PHh//ftPPtbXqlTZz/3v3/OTIJMmk6d/P57n+uQTjrOnHu9rzz81wPo//IAzLhG+ePwxLmNBCft+wiT7Yeo058VHwRj++pvOSMUgBQbzrEBhwHjF6P/8q6+dtyoW3vv4k08jjoHgf77P97g+SwGA3+C3eG/mz2npio0vBwevwzjFY8qPNmjIUHdwDtcJw+DEk8tvu3PGT3Tv8V1+58dpM9x3/gqUTYRRuEyx8Nn7H37sXuOlHtDvIVefHLy+/ZY2dtQRh8ediGgr3z7+8J79RNtfHrysWb1mjX39zbdWtOhedsJxx2SpEEz5Yaot+m2xHV7/MKtYoXz0XSF2XxIZH7xXpkxplwZat04tN0eMi3HW/RmI2M9mrXdGerPjKlvhwkVs0vy1tnhd3jmqapUx+/nu8ras1wHWtE5q+dOZW14b8ZYziokQYdi8+vIgJ8eHDX7OZV3RBrkFxhOZEcuWL3eOyJ73dHYGFscjD/Zyy+J2V4iIDn3xWVfXzJ9AFPD1YYPde1lF01MdxuZ5Z59pbw4fYre0viFPlrlt3LDRfl+y1IoXL2aHHlrXZWXUqVN7h/6LkUukEXlBJl4YlvZhwBMoOvDA1IlY/xP1mQzomESZr25xhb3y0vPOidrprtudTKFOZ8z82Z2H/krUHYO3Qf169sLTA9y5A/r1cdcYP3GSTQ70ZEib9UtgpA5zWRd8lpkcQh8eOvwNp5Nzzs5mZUAy5USGci46N0s+kVvIL68P5ySanui9h4m1rTiki6cee0T/362oEBgm++9X2QlPJvONGzcFylbh6KcR5s5b4DxZdMirmjezl4e/5gZYndq1XDokxvazLwx2aXhhqgYTZddOd1uZ0qXc3xi9eMXmL/jV/Q0MQNbZY3gzme5b8RwXKXjm+RddqolPr2FS8OUAjNiSh0XW++NN7PPo4zY95OkbOer9wFBuEChCbW2vIkVs+YqV9mDw20QfPJxDelanu+6w10e8nZ7KhzPBp336MoUhfevLr79xExEDvO1N19t+lStHPzX3Ovx3XrN12zabMWOmayeEOI4Y7sM7DuKl1996Z8foq4gyQ50vDtorNnWQ74W/61OFfDpk2qzZ1v62tvbV1+Psq2/GuQgbKZY3XXuN62ueeKmYRx95uF15xeVxUzGJCj34SH93va4d78w2Le378RPdvR979FG2fxYTCwbLBx9/4jzpp5x4fCStN0Qi5YxNE4ZwuidKSzg9CkNnzGefB787xhb9Fkmnqh1MQC1bNLOa1Q9ybZQMvu7DqaiUs9qBgeHSuGHQnvXdNSnn0OGv2zUtmzsnE+PrOtbi7bmnvfDSUCsU/H/XHe2sbjCuE70mxLt/T+y9M0ETlR3x9ijn8Mvsmr6PooyQ5vb+R6OdE/GgwLC4pc2N6cpCIuXctHlz+jnxUttwMj75zPMuo8h/vqvbyMP94xBF7hEl4TrxUpCRQSPf/SAo06dO1qKQ1Tq4hrVo1tRqHFTNnUMdTf7hR7d+nCgE93lb29ZurCDPqPu777g1kEWV3PlhEh0fgPysVLGii2Qg88Im+OxV22z272tt39LFrcVhhe3jaXvbbyvW2qTF5azS3hnraO1ms2fG/2VPf7Xc5v+2zEXV6latYE80LG3HVM6XIYXe8963syzft9E/AgZdXy89rb79Rxut36iMhsAdF9W2vudknL9g+fpt1iE4f9TkJW4u2GuvolZz/9LW85xS6WvaV20yO+u5FTb7t9XWr8n+NnzSevt6xuJA5vxl9Q6qaMOalbbqpd2pCbP49yX2zbffu9ctm1+ewTgsXqxYIHNOiP4Vgb7IGuDhr49I7x+Mx+bNmqT3u2TkTVraL06JR761vr6VHVZve3o8/cj3JYiXps4eBScdf5w1uexS9zo3SeTewZfz+GOPCeRwU5eVgLMVvQVjptGlFzkZkCw4a0nZ5vpE2Lje5mBMFSu6Y5YG5yJLJ0ya7GQFyxguC373lJNOSF/b7MtJKu/hwZwaXtfdIrin00852d1TovMxxmO8NNxYGQuJXpOIN3uzbN26ze7r1tlqBPXsQR8kikof7hCcyz2g1yz87nubHhjglQN5Neaz/9natevs7OAefT3Rr4mWE4CZmTbL1SVZnLTvDz9OdTLk4JrVM2RiJFKf/t6R+d0CvXLcd+PdcjV0gvqBzL7lphusZMmITshvTZ02w94c+Y79NDPNXRM9qEmjhnGvmV19emLLGW98+CUTNapXswsD/XhYcP4vgazdmf5JP7nkovOjf0WoVbOmlS1T2pV95arIvkZkTrBfEhkNTYMy+b121gV6DLoMdc/8g26M3Cd74Pxzz3YBii733ucM11ioK86vWLG8uyYZQJnN94mSTDmXLF1mX4/71o0bnCi0z86S6L2Lfx877ar/MDAIiOb645tx30U/+efA6Fy/PhL53meffQKlPfMNNNasWWtPBAotaSgYXWyAh/B58+13nJGO8GFSwhDAAJ4TCKy33hnlPGIoqEyC3khHQTgxEHCkQsVGx5OB6z4/eIgz0pkMUFovb3yZmwTwwJGqBeOCyQUjHYW3+eWN7ebAuGbAn3D8sW6glihRIl3Q+jR2vL4oqbEwoOcvWOBeH1ClSlxFOCum/zQzQz/AaKIddhUo1D0f6GMo4oATpv/Ap9L/zk2I4Nz3wMOBMP/cTWQu02HyD/ZI/wHpBj//9+j14A6pmCi0maVi0r9QclAePopRQGLBcfTF12NdO4Yn5XjM+OlnN7EeHBhBbNAYJiflzA766+NPPmPPvDDYpZMxIXGQwfH4E0+np7XtLJST/t6n3+Ouv3kwADFKGYecg4HOmEaZYhyOHPWea7d4ZHbNrEBhIwIDtEu/xwbawKeedW0J/pqsD4xVLKFv/4FuDwjKRz1x7nMvvpSeyROP2HKi4HpFIF6WDwYuG0IiM0oGyllutZG/f1KgSfvjmpSV8YEzyIMhiaJH1hJGOtAmGCMoFJzvIcJF+9F21FGffo8F/fUtd13a+MPRn0TP3E4y4wPYu8NHxkqWLGkF99jus/5izhZbtmKVHVyxqB0dGNs1KhSzdcG4+TAt4yany4JqP3fQKrtr2FSbvQDj929nBE2aucB6f7YuelbuwGZzxw9caoPG/OTqFtav/9MmB7/d9NlZ9uLkjPuLoCje9FKajZ7wizuPco6fMd/ajVpjm7dE3BQYYRc1apbp0fP+h5x8oA1oQ+bDo488wn03MzAu3v9wtD3w8KMZ+geRLPqDjzAlA+3GdcqXL5/BCEsU+jqOAYxhxkVukZN7R4Y81Le/Gw+cT/9nzDDvJ8uEiZNdWi9ZJlwHcAxy3VgoX5fu97nsPC8rKQtybegrr7lyh8Gw4b68zKM/sM43L+bj7MBgObRuXScXMXzD4CDFSMcpXrtWZPPaM087xf0/aMgwu+r61i59mIjoxeef54xJD85Ozvs1kB0YWoBuMGv2HPe6wWH10o3VZOuTc4a88qpra17zOe03NNAv/bnoC93uu9/JTH9N9CCuOW16zlK3fR8Jl9OPj+69H8yQfg5TAkO0R++H3FwUr38iH5AT8eSHP5AzmcF4RI5CqUAuA8Ew9BZ028qBXuqcX+Mn2n0PPpI+l/y6aJGbA/bZe2+76/Z2zgGdL/iXFaeferKL5udkL414JFPO335bbMuXr7BCexZyjp7rWt/i6ubqG9q4rF7fFsmQzL17KC/OLn77mhtuds4KX1aROuy0oc6awHff/zD9SFTZ3dUsWbbMCQvS03sFA8OnzRCxyWogImz23ru49endw54d2N+tU0LBHReNGFx8wXlG6jZewxtaXe3ew6uJgF4QKCyTpkQUTAR7z3u7BAPlFutwx63uvZwSvu61V7Vwm5Nc0bSRNb7sEvceA5uB7CdJoob1g0ni7DNOs4GPPmwtLm/inA3t2tzovKKAZ7hLhzvd8gAEVCwoxTgtoEjhQpY/OuEgsMNCts2t7Z3HNhYU33A/mDh5im2JoxDkFIQ3KUKsLx3y/NNut2acCz/9nOY+9+n1fn0hRzilh/WHGDfh1EG/tCI2bT82OkkfIULb/ta2NvL1Ydbn/p7u+kxW9AVYESjLGBjUe497OrnrZJeKedopJzkvOwYXmx5mBU9DmDkzzQnhQ+rWjr67I0x0fs0t9xXrlEm0nETJ+Iy6ZuIBIrO+jsLpUWSL4BgAvMOke3E80f+RSMZD/sQmjTC0lV8z6o83hg021h7T9yeGos1ANJr2rl/vUNdXTj35RHu870POY41ShQKRzDX9/fvjnk53OYcYGTWNGl6SrpARLWMtI58RKee+SeNjLNJvUNRjnXY8JYFxSUoibYDyN2/eAlsStAskWs59o9FqIg8oc/0HPGWNmrV0Dj6cZKuD8YxDgV3/c6ONYGxgBODk4f5JgaacXLdv0J8OCGWIfPzJGDdeaI9HHrjPnUNqIHKTe3rvw48yOIloxyf6P+wi8rQnjshe93Z1dbU0aE++EybR8UFUDjnW9va7XNSGcUxb+fZEZI2ZtSHShw4qYoWCt08O/ofx8/90EWrPwHGbbeyPc10k+44La9uqfvXt7ycOt/duP8wO2TeSZnp61fy28v6qtu3JBm5TOOB//vZHeJM6Iue8t/LRBnZE7cwzbB76Yr2lzVvkMrtG3XqYbRnYwKZ2r28NDt7fGeJPjv0jQ1m5H+6xS8M6tnlAffc/TF+01hbuKM6zxEe6kF17Rx1Wvl79wd9AtI4IIE//YDzS7iOCfo+CTBt+8tnnrq8mJW+iRmf5cmWyVbKR5f4aHPx+7+5d3feYO1CWc4tE7z0MBh4ZUYwf5AOOf+TI1OnTo2ckBsbAyPfed3Xl03C5f67rnYwe5gwct+vWrrNbb77JzQeU9aFe3d34IPLnDVPPrpqPgXb1Sx5p88xI9JrICJx1OO2mBLoS9Ql+bqQ+WRqDYQM1a1Z3Dh9kKJmQBGU6391+h2wLjC+cQxg3XpaSoYccwWlVPZrJkZP6JIKPcwYZio7B8hEgndmvk+c1ZT/26KOcnOWeXx70jF195RWBDrjdMZlofXLd10a87frIkYc3cHoR5WTTOeQ0ThfKGoZ2Z9z3D+qd5RY46ijTlB+Sd7jFgmH7RaC/o5+gp5GhAF7e7BPo6ciw0WM+s4f7D3CbG5PRAzhltvyd0fmR1yRTzsVLlri/0eFxnuNwAYxkMsmGxHHm5DbYPej7OFq881ekBjttqJcuVcpFaf2R26lkmYGQI4W832NP2E9RI/24Y45K37UzK84968x0IQsogsuiA+eLr8YGxumd1rpde3v6+UFO+ONlZEBhsPrUEtYCe6PIG7k5BaOX6/JbRJL4bQ7S2gHP3YYNG4P7O9opyEwcd3fuZp279XSR5+zuNx4IlvwFct4d/GZ+/iCanzPVPz4YYu3b3ezWlzIRkTaYl9x847V28onHuzY5sMr+6ZMIj0CDYsWKOgWIiMVDfR9zWQXsEcC5zXGcRJWSMExGzwx81ClRWaW9I7A/D/ohCgAGaLy0RQ9Onh+nTXPXq1tnR4MlJ+XMjj0L7ekmduDRWI898bRN/mGqlQ6MCJQIrxAlC5MaKaPXt7nVKf+Nm19tX40d5z7zyrrnhOOOdesIeZQgSjhOCgxUxiT1ty26MVUy1/SgPBPhZUlIu0Dx8steuO6UKZG0x+ODseiXYdA+555zlqvnJcGEjPc8DBk6zZpc5hTKA6tUsXLlykY/2U4i5SR1G0j1RAYQ2aGPkA6PMsA1ypUtG9RH4VxpI4wNHLVw0gnH2XnBPSOPqAPkqXeEUd5pMyIOXGQtSjHn4KC64Jyz3fvzolFaz1mB8Vy8WHF3HkrwySee4OoTBxP1HU5VT2Z8eJB3PL2iW5cOGaKyv67dZjMW/RnUW2k7bv+IHD96v4JO5sxbstZmLo388uqN2+yjnyJR85PqVLQHzipkJQrnswL5ttn51fPb/Wfs6T7LDZb+uc0mzo8o75ccXt4urJk/kEtmdcvls5uOi6TJLli2zmYtD9eS2U2nV7GepxWyggXy2alVCzoFMkysURt7hI2rRMG4oF0xZtq172AXN77CLmvW0gYPfcV9ThaXn0NzA5R/UpWJRDKG+P0u3Xu5sbE5MMpiHT67kpzcO+Pn5huvc+MH+YDjn7r3QYJEwdjBmEQWXtGkUXr2DU8wiI224ayYNXu2Gz/IhUubtnBl7dC1u5MrfM71wvzT83F2EC1nDiT6zZwIpCbjxKQujjkqkgmCQUKGGZlF1BWGKOfwvw9S8D8wnxxSp5Z7PX7iZOdkIbuDvlStKo8ijjiSclKfQMoyMtRlBQXyjnZnfbOXzX6eGPfd9y4LiQj7X5v/ckEklmwmC/MSuib33bTRpc6JgLwlY+zIBvXdOWTnhZ1JnHPnbW2dwxpZcOghEYefh/dincyxR2wgxEPqPTuVMwZYRoFtEYb6JOX+qecGWZX9KluPrh2t6oF5sx8FY6nFNTe4/hA+vEMyTDLlDDu4OXBIw1djv3GOPk+sI5SD8niHUbKEnTkcOBBZYkZ5yBrGYSJSh5021FlbSZTWHz6Cm9eEDUV2uWaDGdYgxUvzjmWvvTKesykQfgw2wMvEek4OvF4oiRi0KHqkLOcGf0YVcX6L3/S/T1mA3yYCVqfWwdb57jtcSjvnMtmgkDA54FFPBhwsfkJYsnS5m3wAB0TXjne5qFNW+M38/EE0n0G/q2DdUrGYSEBeUrTo9t8muu6VXG80MfnjPScqzXtkFXTq1tOuuPr69LTdnEKqHhkKRBfD6zFjQSklzRHlLxwxCJMb5WSMtb3xemf4UAYUCNYxNg+u+cTTzyXdFwFnWOdAGSGtzqfoZ0V4DJPGXDBQcmNJ9pqAIkdqOE45JlQiRx42pFmzNhKtqVbtQDcuPSUDpQ6lBjmCPAlTqdK+Gc6NJdFykgmEkkUU6NdfF7n2RP5hmCwM5AUZMvRT6iI32ggzELkDB1TZP9N72hzc/+rVkbBt2CEKpUqXcgZ4rNHEfXkwLjBaMiPR8QHME3Dnbbe4yFF4PTN8v3CrLVy6yiqVLmYHlYoYNLUDG6dciaKuL3w6J9KWWwOFln0zoHbFPW3PwPjNK9YF1bR6fSQN/7gqGR0CB5Qq4JSwv/7aYms3ZTTU61Yo6Az6nWWPaDuz+zYOY/DRTu+c8fi5LDcgYyQ2Ih0LS086BgZSOF04r8jJvTPPVa+WsU/mBOQVUdpSJUs4mZ8V1AvjLxn+6fk4O5j7mAORvzz+CplHliKGcviJECzRS/tlttt07fknH3frh8nMHPbqG87IB9/fMWJ5XCCGNLu8/75kSfoeQkceXj99zslJfQK6VlbzwgWBIc+jVtGrKDNL/1rd1Nbu6NDFLUdJlnWBnoCeV7pUSefQ9XCfZA/Apk0bM2RGMufEGtC7AjZaY1kX3NDqKrcPi8enwBOEwmnCXgT3dLrbOS7+iGarZbfENS/ISTnJADvz9FNdm3KQQYPzi/Gbl1Ft+i7jwtc7wUqROuyCaTs1CBuKt93S2ilsCJycgKKD8gikQZHaFT5IL2cNN2lQHr8mPlm2xjGMKkavi4Laof2tO/z+/T27pXs/SRF9LphgSGkjFQ1I72LTkzDb3L/MIQrlI0s4BVi7xORG2p7fpENEIGXs1+gjWnz6MRxcs4Y92ud+t/Mx7YYRwKRNZDQnawyBNmADMCIzpJmVKZ35zk94YFn6QeQ0q7WjuVFO9nfA8GH35+5dOtoRgeJC2dnMjH0sYmESQoG+vOW1rq/FwoZipN5xLyxLISUvniGQDMleE6fF26Pecx5moh3HHHVk9JMIOAT2Lh5xhsyePTeDkwMn3srA4MbgLJbA7qthEi1nsWLFXJQfZwFKKG3AOfPmz7e5c+c5RwIbEXnlL9k24n54zAvp9INeGpqlE4eNsjL7HMUEBQVi5RKpoxgUOFqKJhAJjyWZ8QEsI6IueWxRPEbP2mwbN26wyTPnW+n2kyxfm0lWueNkS5sX2Zjpi182pK/p9sxYvHmH93KTYoUChW+viIH+zfyMBsH0JVuckle4UEErWyy5+S/RNeo4+ejXOEhmzExsqRvz1OtDX3R1Hz7CKe2JUrNGJJtpftDn5mURUWJdvn/0FuPX70zvU6fzil1574lCKjTOLZwF1IOHSHFmmgA6DynNseXkyCwKmsogD5ChRL+Zr1mig7z0T4SI7LcQ2ZeHjdu4/5uuv9ZFizHgWYfP+WHnIn2fwAgBFBwAZALxG/H2StjV9Ykh1/KKy+3VIS+4ZUFs6objkbkiuz1O4sG8xDheEchOnG4e5PjcuZFxxfwW3r8jO5Jdo478/m78hHQjnWj9GaedkkF3xynkHbds8Mc6bAJLlJO5Eohch527uxqyM/ySyfCBg9KTTDm9UU/q+5pg/vN4Jw9jN7ycwTtCwwflySoTM1lwyOCYgQJ7ZO4wEnnPbmOo70pItfQDYPSY/9nKP/5wEyqD7o/VqyPpY4Eg4Ty8X0CqCGlEeCjHfbvjhnpEwL3CzHkcCNiXhg1374XhMXV+beZ7H452/3vnAQYA0TF+n8kCQ4I0ZiYT1sH41NAtWzKuD0cZnjXrFycYOWKhbKSNMhnw/WcHveR2WCdFC6Mqs0e6efweAeGD+8trKD9pvnjO2YEz3r16fHSN/Qyyi7AiQLkW/1PnRC2ZxMlqANoTLzxODtqH3XvJNqF/UJ/LV+y4FpK0pRvb3m7X3nRLpilMfI+JDMWS1PvwBBYL52GwsOQjs0eyJVtOBLaffLi+zzQJwxpADDqMsAL587s1kURv2c0c4nlnUaCpd6LAw98YkUGZBB+NYrddjE3KxoY3Y3dis8pkrklbf/jxGKew4agiJS022sHfRNJhbDDmWa/O9+h7rFFEFvD0ibAzJxESLSfygIg5jgSU0YMDA4b12QXyF7Avx45zv+83hcxJGzF+iUjS5h98NNotkQhDmqPf14AnRnCOHyf8DhttorjRf1BQgI3gSAnmHJSUN0a87d7H+PKpo8mQzPhAYcLhgOOBpyJQhjCklH87J+tN4KYvXGMzgyFSMjCW61SMjIsvpy+2TqM32R8bt9mWbfns/Vlb7eoR6zOsEYfihSLT7bSFa23K7zk37MsUyWdVy0Z2gR85cYm9+zMbA5pN+G2bDfg80o4HV9rbqpXMmaM6O3Da1qlV0/VLls6wYRJ1S33iHAqD0YPSyQZqRJmyMyYSkTf0ceZHPus/8Gl3bX6fgz1yhgx71fU7v/wDyrMEJJgbmT/ffHuUG6O5TbL3vispG9wvBiROG5YF0jZsIsmGkugCYZD95QNdhveHBPoIsiB2bOSUZObjREn0mkQya1av7u6bzXlx7LDJnA86IAcLFYqMI+QcfYNlTSxvQt9C7yKlPLyxLjKKNHf6Ps9yZ76lnb0eCLlVn2+NfNdtjkc/Yqf6Vi2bpztvKUfSWRFlyzjHJmOE+Yr6pJzMY8xnZA4cdtghO8x7uwr/WzxfnH2Wutzd3jlXYmU40X02eaY8zHnAWB/96f/S94dhA+WsZH9ekEw5WTrC+ES//vzLr5wsox2YQ2mH/YL50mc15AX0KbIrcaIgsxrUqxf9RKQCu+Xj2XYWDBg2kfs57Rdn0NwWegQIg5DNmNiQrXwwEZwQGDoj333frUNjg6LM8BFrUpQwsO/s2NW9j4LJAEbB9SA8zz/n7PSd30lv8nDuvV062EHVqrodGlFmUEzw5rHbPNcpW6aM22gHatWs4QxK0qFZZ8vBI23OPftM93kYjE4yCLgu13nzrXfcEQahHW/jKZQBjjBMImGPY16A8DsgEIKsA+O58hwQfhyMh/tFONEmrAX2+MdZheERfGHoB2eedmq6QweBTFSS9PFYKBPrvmLxu74Du77Hqyv/PHTSozD6MgNDzD9DmqUfmU2uyZaTsVCzxkHO8OJ7PvJaudL2x72w/psJl8fKxEJ/zSq6nxm+38a2DZNITknmmqyXw4mGQobT6fIrW0U/yXjvjH/aEaOWXZDDcO9NLrskoTXTYRItJ3+jONI2KGpsOIlMYpJHbtBHWYMOudFGcPaZZ9j4oH5wFmC0cXgYc4Bigrz5NupIurPTPe59D/2OdZbxlitkR6LjA3A2fjn2G6cUjf7kUxdZC0dWpy7ZZouWr7WK5cvaR60r26Hltsu57xdtswufXeB2g5+4aN/gswLW4eS97Ku0ijbn18XW790ZwRE9OSCyEVzGTIpTqxWyt78rYvMW/m71e253fPrHs/lHqU2YkdFpx+Pa+o3CSChhb1x3gNug7vaTitk3aaWcE/WixzKmSGJsdDy1uBUNqnNzEtneyDwiNYlwVYsrAtm11M2PvR56JPrujpC9c3YwDyBnX37lNXeE4RGb4ce7JSJvmN9aXdXCReEwrthBPYzvdyjLZJkxN4fnAsYFzvYwOEq5TqwB7x//FZ4/WC+KYReGv/17fv5I9t4TIV452ZOiSfPIGnZ/TfQBjALm7/C94wTBARiGvy9v3NDVJ3P4dW3aRT+JEG/uTJRE5mOMg9hH6NFmLa+7yb2O1SMSneORj2wqh46EnOBvv8M7IG/4G1nPfeMwjwVHwI/TprvN1oDvkOaO49QvRWQj33Dd5FZ9Ekzw+xvEQsqyNw4TrU/kyfnnnuUeQ0zqOWuew+CgZt+VZOCeWKOeCD+nzXJGOromB8s2w4THfMOLL3I79sfTScksYM6EzMaxr4/wNeON43hjKVGQXYmWE0cjfe+N4BwejcfhYT5mLk9235hk7j2zc5GNl158ocswEamDIuqZgOBjR2b/bGMPhhmbVuENxBi6vMllLlWHDg5EaVEaUbTDcC7PZ2cNqYfnqbKZUayCiWKLACUt2a+FBn6DqBnvMdlccuF5LqKF8YVnDmFHedmAxv8O5W3WuFEGw43NseJ5eJ1CHQjx3j3uSY+Uefg+kdqOd96etADJS5go2txwnVvrg8DLCp6hyiPt2MU0O3yUh3ogVZwdvnFq+Hplje4tbW5wn/n3yMDAGOjd/Z4d+hFkt+s7Xk4eOeUUjjjPQw/jH8kWjhjEI9lyck7zpk2c48pnIMTC94lCsNzE1znnkvJJqnU8oY+ji/VZnNes8WU7KI/0PzY3Ic0QUM6aNWlkt7dr4/7OCclck3R8xlN2EPHu2a1zMEFf6OoBqAMyFfo//EC6gpcMiZaT3/EbDBHFYaxTn8gIYD2hlx85aSPKgTyg/513zlnO8IgFo/CB++515Stfrpx7z9//tVe3TFdG6Vd9evVwfd73I/5HEer3YO8Ma/8TJZnxASzj4fnAnH/Wmafv0Oc+SNvsduivVqH4DtHo6mXy2f5li7ko28hpPFbPrFbQPN/dWsFanVHLKkbbgR3gT6hX1QZcUspF3cO0Oryg3XPxQVapQqSedoZTquSzT9tWsTOOqBYo6BFnA/83PL6Gjb3tADurWu5O7a497+/hNqHy7Q6sX2U+9PKMNmG5QdvWN7gooJ8nMyMReQOMK8YXcivcn1hrjIJNv+NapAoja3jNUe+Qum7ZR+wmWLlBsve+K2Eu51nOjRte4uqFspBFw9yOIyQW9B3kQFg+7AqSmY8TJZlr+k3lgMeV1q4dyYDzIPe4b/oFdQTISvowe5IAwQ10Jg8BF2QyoOd54ytMbtRn00YNnbz08wzlZRzSv1pc0TS9/MmAfGc84NDy30cn4Zo4EzA+c4v1GzYmNMcCji/mWd9OjCXGFHrYdVdfmaN7zw0SLSfjk0cus3Gk1z/pJ/QX+o1fK55XICOQpSyhxUmQKvUpIuQLDLZdk+O0m0L1sCaZNDrSrTITXCiNpB4huKdO/yndixXrlcOo5noMYCJtDNis4PdJC966ZasVLlI4XfEN43+btMHMrkkq4MZAMLIRHgpqdr8L/jvAfeUk4vVvhvvv07e/87LGi7LnNlOnTXfPLEXB4HExmfU9IoQ8m5vIARsoHnt0xrXUQuyOJDo+Euap5KJHuxWtJ0ZfCLH7QKZZ914POic2AQwciEII8W9CEfVswKDFsCVdJCtFkM84JztjFk8VEelEjWXOIT2Ta8cz0sH/dlbX5Lucw7US+V3w30nkvsSuBeOb/Qn4/+QTjs+y7/38c5pNnDTZRQ6IIAixu5PM+BBC/Pdgze/QV15zy5LI3mLHdiGE+LehiHouEF4jlJN1aCI1+Kcj6kKIPEQRdSH+9cSuvyVNvVunu3O0vEYIIf5pFFHPBXguJRtHcGS3wZEQQgghhNh1kL3IWuEeXTvJSBdC/GtRRF0IIYRQRF0IIYQQKYQMdSGEEEIIIYQQIoVQ6rsQQgghhBBCCJFCyFAXQgghhBBCCCFSCBnqQgghhBBCCCFECiFDXQghhBBCCCGESCFkqAshhBBCCCGEECmEDHUhhBBCCCGEECKFkKEuhBBCCCGEEEKkEDLUhRBCCCGEEEKIFEKGuhBCCCGEEEIIkULIUBdCCCGEEEIIIVIIGepCCCGEEEIIIUQKIUNdCCGEEEIIIYRIIWSoCyGEEEIIIYQQKYQMdSGEEEIIIYQQIoWQoS6EEEIIIYQQQqQQMtSFEEIIIYQQQogUQoa6EEIIIYQQQgiRQshQF0IIIYQQQgghUggZ6kIIIYQQQgghRAohQ10IIYQQQgghhEghZKgLIYQQQgghhBAphAx1IYQQQgghhBAihZChLoQQQgghhBBCpBAy1IUQQgghhBBCiBRChroQQgghhBBCCJFCyFAXQgghhBBCCCFSCBnqQgghhBBCCCFECiFDXQghhBBCCCGESBnM/g9F5fJOKCTAmQAAAABJRU5ErkJggg==">

Reply:
@vadgun not sure what the question is here but I am happy with how it currently is including the `sts:GetCallerIdentity` permission